core/src/banking_stage.rs

```diff
@@ -569,6 +576,77 @@ impl BankingStage {
             ));
         }
 
+        let transaction_struct = if enable_forwarding {
+            warn!(
+                "Forwarding only supported for `Sdk` transaction struct. Overriding to use `Sdk`."
+            );
+            TransactionStructure::Sdk
+        } else {
+            transaction_struct
+        };
+
+        match transaction_struct {
+            TransactionStructure::Sdk => {
+                let receive_and_buffer = SanitizedTransactionReceiveAndBuffer::new(
+                    PacketDeserializer::new(non_vote_receiver),
+                    bank_forks.clone(),
+                    enable_forwarding,
+                );
+                Self::spawn_scheduler_and_workers(
```

Comment:
we could if we had a `dyn` instead of `impl`, but then it'll have some small additional overhead on each call. and I'd rather just duplicate the few lines at that point.